src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -1148,6 +1148,63 @@ impl ToQemuParams for Rtc {
     }
 }
 
+#[derive(Debug)]
+struct ObjectRngRandom {
+    // id is the device ID
+    id: String,
+
+    // filename is the entropy source on the host
+    filename: String,
+}
+
+impl ObjectRngRandom {
+    fn new() -> ObjectRngRandom {
+        ObjectRngRandom {
+            id: "rng0".to_owned(),
+            filename: "/dev/urandom".to_owned(),
+        }
+    }
+}
+
+#[async_trait]
+impl ToQemuParams for ObjectRngRandom {
+    async fn qemu_params(&self) -> Result<Vec<String>> {
+        let mut object_params = Vec::new();
+
+        object_params.push("rng-random".to_owned());
+        object_params.push(format!("id={}", self.id));
+        object_params.push(format!("filename={} ", self.filename));
```

Comment:
```suggestion
        object_params.push(format!("filename={}", self.filename));
```

Reply:
This should address https://github.com/kata-containers/kata-containers/pull/9639#issuecomment-2117562446.